hack/tfconfig.yaml

```diff
@@ -52,11 +52,14 @@ configs:
 
 modifiers:
   "01000000":
+    priority: 0
```

Comment:
exclusive modifier only set 
UseSubtree = true (https://github.com/kubewharf/kelemetry/blob/main/pkg/frontend/tf/defaults/modifier/exclusive.go#L65) , SetRoot is really run at https://github.com/kubewharf/kelemetry/blob/main/pkg/frontend/tf/transform.go#L85, i mean that ProcessExtensions in transform can be run after SetRoot